Fabric.Identity.API/scripts/Install-IdPSS-Utilities.psm1

```diff
@@ -262,6 +271,24 @@ function Clear-IdentityAppSettings {
 
 }
 
+function Clear-IdentityEnvironmentSettings {
```

Comment:
Is this function appropriately named? Seems like we are only clearing the azure settings?